k8s/kube-base/job-xdmod-openshift-staging.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: xdmod-openshift-staging-job
+spec:
+  schedule: "0 1 */1 * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: xdmod-openshift-staging-job
+              image: image-registry.openshift-image-registry.svc:5000/xdmod-tzumainn/moc-xdmod:latest
+              imagePullPolicy: IfNotPresent
+              envFrom:
+              - secretRef:
+                  name: xdmod-openshift-staging-secret
+              env:
+              - name: OPENSHIFT_PROMETHEUS_URL
+                value: https://prometheus-k8s-openshift-monitoring.apps.ocp-staging.massopen.cloud
+              - name: OPENSHIFT_SERVER_URL
+                value: https://api.ocp-staging.massopen.cloud:6443
+              - name: OUTPUT_DIR
+                value: /root/xdmod_openshift_data/nerc_openshift_staging
+              - name: XDMOD_RESOURCE
+                value: nerc_openshift_staging
+              command: ["/usr/bin/xdmod-openshift-reporting.sh"]
+              volumeMounts:
+                - name: vol-xdmod-conf
+                  mountPath: /etc/xdmod
+                - name: vol-xdmod-src
+                  mountPath: /usr/share/xdmod
+                - name: vol-xdmod-init
+                  mountPath: /mnt/xdmod_init
+                - name: vol-var-log-xdmod
+                  mountPath: /var/log/xdmod
+                - name: vol-xdmod-openshift-data
+                  mountPath: /root/xdmod_openshift_data
```

Comment:
Let's use a different directory other than root for this.  Let's create a /data directory as previous review comments view the /root directory as special.

Reply:
Sounds good - done!